Resources/doc/index.rst

```diff
@@ -122,6 +130,15 @@ the MongoDB ODM across your application:
             default:
                 auto_mapping: true
 
+.. note::
+
+    If you are using Symfony Flex, you can allow `recipes` in the "contrib" repository to 
+    work with this bundle by executing the following command:
+
+.. code-block:: bash
+
+    composer config extra.symfony.allow-contrib true
```

Comment:
Question: how does `.. code-block` work with a `.. note`? In context, I imagine this code block should reside within the note, but I expect it will actually render outside.

Based on [this RST preview](http://rst.ninjs.org/?theme=nature#Li4gbm90ZTo6CgogICAgSWYgeW91IGFyZSB1c2luZyBTeW1mb255IEZsZXgsIHlvdSBjYW4gYWxsb3cgYHJlY2lwZXNgIGluIHRoZSAiY29udHJpYiIgcmVwb3NpdG9yeSB0byAKICAgIHdvcmsgd2l0aCB0aGlzIGJ1bmRsZSBieSBleGVjdXRpbmcgdGhlIGZvbGxvd2luZyBjb21tYW5kOgoKICAgIC4uIGNvZGUtYmxvY2s6OiBiYXNoCgogICAgICBjb21wb3NlciBjb25maWcgZXh0cmEuc3ltZm9ueS5hbGxvdy1jb250cmliIHRydWU=) it looks like we can resolve this by aligning the indent of `.. code-block:: bash` with the paragraph above ("If you are using Symfony Flex..."). The `composer config` line itself will then need to be indented further.